nero_view.py

```diff
@@ -89,7 +89,7 @@ def check_can(self):
 
     def update_current_page(self):
         self.current_screen.current_page.update()
-        self.after(100, self.update_current_page)
+        self.after(250, self.update_current_page)
```

Comment:
I want to make this 50 ideally aka 20 fps, buttttttt I cant get it to work without tanking the pi, even without graphs

Reply:
rip